core/src/main/java/org/radargun/stats/Statistics.java

```diff
@@ -54,6 +55,24 @@
    Statistics copy();
 
    /**
+    * Creates a logical group of operations identified by given group name. Should be used on per-stage basis.
```

Comment:
When this is on a per-stage basis, why do you add it to Statistics which have multiple instances in each stage (typically one for each thread)? Why can't you put this just to the stage itself? It's set to report from acks anyway.